sunpy/timeseries/timeseries_factory.py

```diff
@@ -50,7 +51,7 @@ class TimeSeriesFactory(BasicRegistrationFactory):
         Inputs to parse for timeseries objects. See the example section for a
         detailed list of possible inputs.
 
-    source : `str`, optional
+    source : {Source}
```

Comment:
`({‘eve’, ‘noaaindices’, ‘norh’, ‘rhessi’, ‘gbmsummary’, ‘lyra’, ‘xrs’, ‘esp’, ‘noaapredictindices’})`

This lacks the `|`  required as per the numpydoc style.